### ISSUE TITLE
Add Expired tag to expired posts on View Profile page

### ISSUE THREAD

**Issue by vikjoshi** [FightPandemics/FightPandemics#930]
As an individual user or organizational user, I can see all posts that i have created under My Activity on the View Profile page. For posts that have expired, can we add an Expired tag to differentiate between active and inactive posts? 

Need some Design input so tagging @Yu-Alexa-Zhao here for now. 

To be prioritized for post soft-launch

**Comment by Yu-Alexa-Zhao** [comment]
We were working previously under the assumption that expired posts will not show up in the feed. What are the benefits of showing them in the feed?

**Comment by joshmorel** [comment]
@Yu-Alexa-Zhao they won't show up in the feed. But the idea is to show them in under organization/user activity.

@vikjoshi  maybe displaying the expiry date for all posts somewhere would be helpful (for more than just this) and if it's expired it's highlighted red or something like that?

I can add a param - e.g. `includeExpired` once we are ready with a design.

**Comment by doheekim93** [comment]
@anjanarao24 Hey Anjana, are you still leading this "adding new sections in profile page" feature pod?

**Comment by emtay** [comment]
@dhruvdpatel @tejasm2  here is ticket to show expired tag. leverage this for your updates? (will be separate from the "section" in the profile page that Do Hee mentioned)

should it be P2? would be important for people or organizations to know that their posts are no longer being found because they expired

**Comment by tejasm2** [comment]
> @dhruvdpatel @tejasm2 here is ticket to show expired tag. leverage this for your updates? (will be separate from the "section" in the profile page that Do Hee mentioned)
> 
> should it be P2? would be important for people or organizations to know that their posts are no longer being found because they expired

Yes, I think we can simply indicate the expiry labels which user selects. For example, "Expires in a day", "Expires in a Week", "Expires in a Month", "Never Expires"....something like that...may be design team can propose. I think displaying these tags on the posts should not be that difficult. If we are displaying the timestamp of when the post is created as well as these labels, then it should be fine for launch. Maybe storing or displaying the expired posts in the profile we can look at after launch?
